includes/html/print-interface.inc.php

```diff
@@ -82,11 +82,11 @@
 
 if ($port_details) {
     $port['graph_type'] = 'port_bits';
-    echo generate_port_link($port, "<img src='graph.php?type=port_bits&amp;id=" . $port['port_id'] . '&amp;from=' . Config::get('time.day') . '&amp;to=' . Config::get('time.now') . '&amp;width=100&amp;height=20&amp;legend=no&amp;bg=' . str_replace('#', '', $row_colour) . "'>");
+    echo generate_port_link($port, "<img src='graph.php?type=port_bits&amp;id=" . $port['port_id'] . '&amp;from=' . Config::get('time.day') . '&amp;to=' . Config::get('time.now') . '&amp;width=100&amp;height=20&amp;legend=no&amp;bg=' . str_replace('#', '', $row_colour) . "00'>");
```

Comment:
how does this affect light theme?

Reply:
seems like it always makes the color 100% transparent

Reply:
Sorry for the delay, im really busy during the week, it always make the color 100% transparent its a upgrade even to the light, blue and mono theme, it looks like in all other themes, the background is always white (screenshot1), it just changes the behavior when you do hover over the port by itself... see screenshot 2


screenshot1
![image](https://user-images.githubusercontent.com/7550822/129994411-d8e82327-5934-47c4-897e-ac0e93c79041.png)

Screenshot2 (this will now be transparent so the rectangles will not be visible)
![image](https://user-images.githubusercontent.com/7550822/129994428-4af3f0ab-ba21-452e-bcf9-30439190a2fc.png)